book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] It seems like you might have forgotten to implement the "many-to-many (also has-many-through) relationship between Person and Book using the intermediate class Rental"  the a book has many rentals association.

![image](https://user-images.githubusercontent.com/86318284/195912326-78168a6a-2f5e-43a6-955c-48f9ecbac84b.png)

Reply:
Hi @Nemwel-Boniface 
I indeed implemented the many-to-many relationship using the intermediate class Rental as requested by the project requirements. So when a new Rental instance is created, the initialization takes action to assign the rental to the person and the book declared. You can try running the following code:
```
require_relative './person'
require_relative './book'
require_relative './rental'

andy = Person.new(33, 'Andy')
cindy = Person.new(33, 'Cindy')

book1 = Book.new('The Iliad', 'Homer')
book2 = Book.new('The Complete Stories', 'Isaac Asimov')

rental01 = Rental.new('2022-10-12', book1, andy)
rental02 = Rental.new('2022-10-12', book2, andy)
rental03 = Rental.new('2022-08-10', book1, cindy)
rental04 = Rental.new('2022-08-12', book2, cindy)

puts andy.rentals.map { |rental| rental.book.title }
puts cindy.rentals.map { |rental| rental.book.title }
puts book1.rentals.map { |rental| rental.person.name }
puts book1.rentals.map { |rental| rental.person.name }
```
There you will see that it is possible to have "many-to-many" relationship between Person and Book